tfhe/src/core_crypto/gpu/mod.rs

```diff
@@ -19,6 +19,12 @@ use std::ffi::c_void;
 use tfhe_cuda_backend::bindings::*;
 use tfhe_cuda_backend::cuda_bind::*;
 
+pub enum PBSMSNoiseReductionType {
+    NoReduction = 0,
+    Drift = 1,
+    Centered = 2,
+}
```

Comment:
I think you shouldn't need to have this enum on the Rust side, you could reuse PBS_MS_REDUCTION_T by binding it between Rust & C